packages/dso-toolkit/src/styles/components/_dropdown.scss

```diff
@@ -20,6 +20,8 @@ $dso-dropdown-hor-padding: 20px;
   }
 
   .dropdown-menu {
+    box-shadow: 0 8px 10px 1px rgba(0, 0, 0, 0.4);
```

Comment:
Inmiddels zie ik wel vrij veel verschillende box-shadows:
- card, dso-select, whitebox ...
En dan heb ik het nog niet over wat uit bootstrap komt... Daarover: je gaat hier OVER die van de bootstrap `.dropdown-menu` heen. Halen we 'm niet liever uit die nu ingecheckte vorm weg? Of, ... wat is daarvoor nu eigenlijk onze strategie, @tfrijsewijk ?

Reply:
Die verschillende box-shadows storen mij ook, misscheon maar eens bij Bram aankaarten?

Reply:
Is een issue voor aangemaakt. https://github.com/dso-toolkit/dso-toolkit/issues/963

Reply:
Issue voor aangemaakt: https://github.com/dso-toolkit/dso-toolkit/issues/963